backend/src/main/java/ubb/courses/backend/services/CourseService.java

```diff
@@ -0,0 +1,50 @@
+package ubb.courses.backend.services;
+
+import org.springframework.stereotype.Service;
+import ubb.courses.backend.DTOs.CourseDTO;
+import ubb.courses.backend.DTOs.LessonDTO;
+import ubb.courses.backend.models.Course;
+import ubb.courses.backend.repositories.CourseRepository;
+
+import java.util.Set;
+import java.util.stream.Collectors;
+
+@Service
+public class CourseService {
```

Comment:
Please create an interface and implement it here and leave the methods empty for implementing in the future prs